ci-operator/step-registry/hypershift/kubevirt/baremetalds/redhat-operators/hypershift-kubevirt-baremetalds-redhat-operators-commands.sh

```diff
@@ -49,16 +49,14 @@ function mirror_ccs() {
     echo "3: Check skopeo and registry credentials"
     if [[ ! -f /usr/bin/skopeo ]]; then
         yum install -y skopeo
-        oc -n openshift-config extract secret/pull-secret --to="/tmp" --confirm
-        set +x
-        mirror_token=$(cat "/tmp/.dockerconfigjson" | jq -r --arg var1 "${mirror_registry}" '.auths[$var1]["auth"]'|base64 -d)
-        skopeo login "${mirror_registry}" -u "${mirror_token%:*}" -p "${mirror_token#*:}"
-        REGISTRY_REDHAT_IO_USER=$(cat /home/pull-secret | jq -r '.auths."registry.redhat.io".auth' | base64 -d | cut -d ':' -f 1)
-        REGISTRY_REDHAT_IO_PASSWORD=$(cat /home/pull-secret | jq -r '.auths."registry.redhat.io".auth' | base64 -d | cut -d ':' -f 2)
-        skopeo login registry.redhat.io -u "${REGISTRY_REDHAT_IO_USER}" -p "${REGISTRY_REDHAT_IO_PASSWORD}"
-        set -x
     fi
-
+    oc -n openshift-config extract secret/pull-secret --to="/tmp" --confirm
+    set +x
+    mirror_token=$(cat "/tmp/.dockerconfigjson" | jq -r --arg var1 "${mirror_registry}" '.auths[$var1]["auth"]'|base64 -d)
+    skopeo login "${mirror_registry}" -u "${mirror_token%:*}" -p "${mirror_token#*:}"
```

Comment:
isn't the username and password base64-encoded?
anyway, can't you login directly with the authfile you've extracted in an earlier step? i.e
```
skopeo login "${mirror_registry}" --authfile /tmp/.dockerconfigjson
```

Reply:
Hi @orenc1 , I will recover the original `skopeo login` steps , you can see oc-mirror v2 no flag `--oci-registries-config` 
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/69346/rehearse-69346-periodic-ci-openshift-hypershift-release-4.19-periodics-e2e-kubevirt-metal-ovn-disconnected/1972553077969915904